git-link.el

```diff
@@ -360,11 +360,11 @@ Defaults to \"origin\"."
 
   (interactive (list (git-link--select-remote)))
   (let* ((remote-host (git-link--remote-host remote))
-	 (commit      (word-at-point))
+	 (commit      (downcase (word-at-point)))
```

Comment:
Ah, keep on missing the non-string case. Yeah, I was thinking of adding `A-F` to the regexp instead, will do that now. 

>  will one encounter an uppercase letter in a SHA?

I haven't encountered that. But I think it should be supported as uppercase hex is legal too.